cypress/e2e/deputies/tab_visibility.cy.js

```diff
@@ -0,0 +1,45 @@
+describe("Deputies visibility", () => {
+    before(() => {
+        cy.setCookie("Other", "other");
+        cy.setCookie("XSRF-TOKEN", "abcde");
+    })
+
+    let teams = [24, 27] // PA and Pro
```

Comment:
Instead of adding a comment, if you name the variable to `PaProTeams`, the code documents itself.

Reply:
Good point will change